experimental/traffic-portal/nightwatch/dataClient.ts

```diff
@@ -40,7 +40,7 @@ import {
 	RequestType,
 	ResponseCacheGroup,
 	ResponseDeliveryService,
-	ResponseDivision,
+	ResponseDivision, ResponseParameter,
```

Comment:
please put this on its own line, in keeping with the rest of the import statement